tech-far-hub/src/components/page-layout-nav.tsx

```diff
@@ -12,6 +12,14 @@ interface IPageLayoutNav {
   siblings: {
     readonly nodes: readonly {
       readonly frontmatter: IMinimalFrontmatter | null;
+      readonly parent:
+        | {}
```

Comment:
Assuming the pipe is an OR, if so, why is that leading OR needed in the definition of the readonly parent?

Reply:
You know, I don't know --- the formatted just adds it when I hit save. Imma have to figure that out...